README.md

```diff
@@ -109,17 +109,45 @@ You can also cross-compile if necessary:
 GOOS=windows GOARCH=amd64 make build
 ```
 
+#### Unit tests
+
 In order to run the Unit Tests for the provider, you can run:
 
 ```sh
 $ make test
 ```
 
-The majority of tests in the provider are Acceptance Tests - which provisions real resources in Azure. It's possible to run the entire acceptance test suite by running `make testacc` - however it's likely you'll want to run a subset, which you can do using a prefix, by running:
+With VSCode Golang extension you can also run and debug the tests using `run test`, `debug test` `run package tests`, `run file tests` buttons.
 
-```sh
-make testacc SERVICE='resource' TESTARGS='-run=TestAccAzureRMResourceGroup' TESTTIMEOUT='60m'
-```
+#### Acceptance tests
+
+The majority of tests in the provider are acceptance tests - which provisions real resources in Azure Devops and Azure. To run any acceptance tests you need to set `AZDO_ORG_SERVICE_URL`, `AZDO_PERSONAL_ACCESS_TOKEN` environment variables, some test have additional environment variables required to run. You can find out the required environment variables by running the test.
```

Comment:
In my perspective,  developers who want contribute to this provider,  mostly they focus on the resource they interested.  `make testacc` will run all acceptance tests, this will get developers confused with the error messages.

Reply:
Error message? I'm just updating the existing documentation since it was out of date. I think documenting why the acceptance tests will fail if you run all of them is more than valid. I know I was confused for a while :D

Reply:
make testacc was previously mentioned as well

Reply:
I'll be merging this PR, because it's gotten a bit out of hand, if this is still an issue I can make a new PR to address this :)